pkg/agent/containerd/nvidia_test.go

```diff
@@ -0,0 +1,164 @@
+package containerd
+
+import (
+	"io/fs"
+	"testing"
+	"testing/fstest"
+
+	"github.com/rancher/k3s/pkg/agent/templates"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFindNvidiaContainerRuntimes(t *testing.T) {
```

Comment:
Was this test generated based on the doc here? https://github.com/k3s-io/k3s/blob/master/tests/TESTING.md

Reply:
Tests have been regenerated with gotests tool :)